photo-points/Controllers/AdminController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using photo_points.Models;
+using photo_points.Controllers;
+//using photo_points.ViewModels;
+
+// For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace photopoints.Controllers
+{
+    public class AdminController : Controller
+    {
+
+
+        private PhotoDataContext _pdc;
+
+        public AdminController(PhotoDataContext pdContext)
+
+        {
+            _pdc = pdContext;
```

Comment:
Use the IRepository for the UserRepository instead of the data context here

Reply:
I talked to Eric he is working on it, I will delete it on my next step (set visual studio for pc user and mac user)